src/Writers/HTMLWriter.jl

```diff
@@ -740,6 +746,8 @@ function mdconvert(link::Markdown.Link, parent)
         path = first(s)
         path = endswith(path, ".md") ? Formats.extension(:html, path) : path
         url = (length(s) > 1) ? "$path#$(last(s))" : Compat.String(path)
+        # Replace any backslashes in links, if building the docs on Windows
+        url = replace(url, '\\', '/')
```

Comment:
who's the "user" here? a lot of these come from file paths which have the backslashes in them